### PR TITLE
Add parameter to make setting actions/locks easier

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
This adds a class parameter to make it easier to set locks/actions via an ENC.

#### This Pull Request (PR) fixes the following issues
None

### CODE REVIEW

**Comment by bastelfreak** [comment]
thanks for the PR, can you please run `bundle exec rake rubocop:auto_correct`?

**Comment by jcpunk** [comment]
I'm showing complaints about new cops needing a default setting rather than existing code complaints.

File: manifests/plugin/post_transaction_actions.pp [hunk 1]
```diff
@@ -39,5 +45,7 @@
       content => "# Puppet maintained actions\n# \$key:\$state:\$command\n\n",
       order   => '01',
     }
+
+    create_resources('yum::post_transaction_action', $actions)
```
Comment (bastelfreak) [inline manifests/plugin/post_transaction_actions.pp:49]: please don't use the legacy create_resources. Instead iterate on the hash and use the splash operator for the resources. An example is here: https://github.com/voxpupuli/puppet-ferm/blob/master/manifests/init.pp#L88-L92

Comment (jcpunk) [reply manifests/plugin/post_transaction_actions.pp]: Done

**Comment by jcpunk** [comment]
It looks like the dnf support for versionlock is not entirely complete... mostly it seems to be putting the lockfile in the wrong place.

**Pull request closed without merging**